docs/blog/posts/stable-release-announcement.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2024-03-10
+authors: [agardnerIT]
+description: >
+  In this blog post you will learn how to configure Keptn via the new Helm umbrella chart.
+comments: true
+---
+
+# Announcing the Official Release Candidate for Keptn v2
+
+The Keptn project is proud to announce a release candidate for what will become Keptn v2.
+We are sharing this release candidate
+so that the community and end users can participate in the testing
+to ensure that the actual release is as robust as possible.
+We anticipate (as long as no release blockers are identified)
+that Keptn v2 will be released approximately 1 month after this announcement.
+
+## What Problem Does Keptn Solve?
+
+Deploying software using Kubernetes is, on the surface, easy.
+Just use kubectl or a GitOps solution like ArgoCD to deploy a YAML file and that’s it, right?
+Well, no.
+Not normally.
+There is a lot more to ensuring a healthy deployment.
+The pod(s) may be running, but that doesn’t automatically mean that the application is healthy.
+This is the problem domain that Keptn acts upon.
+By using Keptn, in combination with your standard
+deployment tooling or practices, you can move from “I guess it’s OK” to “I know it’s OK”.
+Keptn allows you to wrap governance and automated checks around the deployment process to ensure that
+the end-to-end process of deploying is healthy and your application is meeting the SLOs you’ve defined.
+
+## What’s New in Keptn v2?
+
+User feedback to the Keptn project has been clear, and we’ve listened.
```

Comment:
Is this sentence really necessary and does it add anything?

Reply:
Okay, but maybe not as the topic sentence for this section.  I'm rewording and moving this up to the beginning of the blog.